Pear/Controller/HomeViewController.swift

```diff
@@ -239,4 +247,33 @@ extension HomeViewController: UICollectionViewDelegateFlowLayout {
         let cellWidth = indexPath.item == 0 ? 151 : 50
         return CGSize(width: cellWidth, height: 40)
     }
+} 
+
+extension HomeViewController: UNUserNotificationCenterDelegate {
+    private func setupLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            if granted {
+                center.delegate = self
+                // get rid of previously scheduled notifications
+                center.removeAllDeliveredNotifications()
+                center.removeAllPendingNotificationRequests()
+                self.scheduleNotifications(center: center, day: 2, hour: 8)
+                self.scheduleNotifications(center: center, day: 4, hour: 14)
+                self.scheduleNotifications(center: center, day: 6, hour: 12)
+            }
+        }
+    }
+
+    private func scheduleNotifications(center: UNUserNotificationCenter, day: Int, hour: Int) {
+        let content = UNMutableNotificationContent()
+        content.title = "Meet your new pear!"
+        content.body = "Set up this week's chat today 😊"
```

Comment:
We are scheduling three notifications throughout the week but I think the message we show for the first day is different than the others right? So the first day notifies users of a match but then the next two remind them to check the app? Can you double check with Cindy?